eclair-core/src/test/scala/fr/acinq/eclair/FeaturesSpec.scala

```diff
@@ -41,12 +41,28 @@ class FeaturesSpec extends FunSuite {
   }
 
   test("'variable_length_onion' feature") {
-    assert(hasFeature(hex"0100", Features.VARIABLE_LENGTH_ONION_MANDATORY))
+    assert(hasFeature(hex"0100", VARIABLE_LENGTH_ONION_MANDATORY))
     assert(hasVariableLengthOnion(hex"0100"))
-    assert(hasFeature(hex"0200", Features.VARIABLE_LENGTH_ONION_OPTIONAL))
+    assert(hasFeature(hex"0200", VARIABLE_LENGTH_ONION_OPTIONAL))
     assert(hasVariableLengthOnion(hex"0200"))
   }
 
+  test("'option_static_remotekey feature") {
+    val optionalSupport = hex"2000"
+    val mandatorySupport = hex"1000"
+    val noSupport = hex"0000"
+
+    assert(hasFeature(mandatorySupport, STATIC_REMOTEKEY_MANDATORY))
+    assert(hasFeature(optionalSupport, STATIC_REMOTEKEY_OPTIONAL))
+
+    assert(Features.canUseStaticRemoteKey(localFeatures = optionalSupport, remoteFeatures = mandatorySupport) == true)
```

Comment:
Although generally ugly I found more readable to explicit the `== true` when the assertion condition is very long, mostly because the negation is hard to spot with naked eyes. Not a big deal though, i can remove it.